release_build_files/readme.md

```diff
@@ -1717,4 +1718,4 @@ code.
         [setup guide](https://firebase.google.com/docs/cpp/setup) to get
         started.
 -   Known Issues
-    -   Android armeabi libraries must be linked with gcc 4.8.
\ No newline at end of file
+    -   Android armeabi libraries must be linked with gcc 4.8.
```

Comment:
(is this just a newline?)

Reply:
Ugh yes. It gets added and I keep manually removing it. Maybe it's time to let it win